src/main/java/org/sonatype/nexus/ci/nxrm/v3/MoveComponentsStep.java

```diff
@@ -98,6 +98,7 @@ public void perform(@Nonnull final Run run, @Nonnull final FilePath workspace, @
     catch (RepositoryManagerException e) {
       listener.getLogger().println("Failing build due to: " + e.getResponseMessage().orElse(e.getMessage()));
       run.setResult(FAILURE);
+      throw new IOException(e);
```

Comment:
If you comment out this line, the tests still pass. They probably shouldn't unless they aren't for testing this case (I assumed they were)

Reply:
Hmmm..looking now.

Reply:
The workflow test behaves differently than expected, still looking at that one. Thanks for the good eyes @nmcafee!